jenkins/scripts/skip_integration_test.sh

```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env bash
+
+set -eu
+
+exclude_md() {
+  for file in $(git diff "${ghprbActualCommit}" origin/"${ghprbTargetBranch}" --name-only)
+  do
+    filename=$(basename -- "$file")
+    extension="${filename##*.}"
+    filename="${filename%.*}"
+    if [[ $extension != "pipeline" ]]
+    then
+      if [[ $extension != "sh" ]]
+      then
+        echo "The file(s) changed contains other extensions and files than pipeline or shell script"
+        return 1
```

Comment:
I did not understand this. Why are we checking for pipeline and sh? I thought we would check if all changed files are markdown, then skip.

Reply:
That is for testing purposes and verify this patch is working, since we are changing .pipeline and .sh files in this parch, I am using those to test. See the previous [comment](https://github.com/metal3-io/project-infra/pull/305#issuecomment-1048657961) on this PR.